backend/src/schema/types/type/REPORTED.gql

```diff
@@ -0,0 +1,42 @@
+type REPORTED {
+  createdAt: String
+  reasonCategory: ReasonCategory
+  reasonDescription: String
+  submitter: User
+    @cypher(statement: "MATCH (resource)<-[:REPORTED]-(user:User) RETURN user")
+  # not yet supported
+  # resource: ReportResource
+  #   @cypher(statement: "MATCH (resource)<-[:REPORTED]-(user:User) RETURN resource")
+  type: String
+    @cypher(statement: "MATCH (resource)<-[:REPORTED]-(user:User) RETURN labels(resource)[0]")
+  user: User
+  post: Post
+  comment: Comment
+}
+
+# this list equals the strings of an array in file "webapp/constants/modals.js"
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-10-16T07:10:48Z" title="Wednesday, October 16th 2019, 9:10:48 am +02:00">Oct 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,42 @@
+type REPORTED {
+  createdAt: String
+  reasonCategory: ReasonCategory
+  reasonDescription: String
+  submitter: User
+    @cypher(statement: "MATCH (resource)<-[:REPORTED]-(user:User) RETURN user")
+  # not yet supported
+  # resource: ReportResource
+  #   @cypher(statement: "MATCH (resource)<-[:REPORTED]-(user:User) RETURN resource")
+  type: String
+    @cypher(statement: "MATCH (resource)<-[:REPORTED]-(user:User) RETURN labels(resource)[0]")
+  user: User
+  post: Post
+  comment: Comment
+}
+
+# this list equals the strings of an array in file "webapp/components/Modal/ReportModal.vue"
```


This file has changed to `webapp/constants/modals.js` …
I’ll push this up now …